src/encoded/memlimit.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, application, callback):
     def __call__(self, environ, start_response):
         try:
             result = self.__application(environ, start_response)
-        except:
+        except BaseException:
```

Comment:
why BaseException here, and not Exception?

Reply:
The precise meaning of a raw `except:` is `except BaseException:`. Doing it here indicates I could not prove to my own satisfaction that it was safe to rewrite this as `except Exception:`, which is a semantic change but often what most programmers _thought_ they were saying when they wrote `except:`. So I go for bug-for-bug compatibility and explicit semantics in the uncertain case. There are only a few of these. We might want to do some empirical tests later to clean this up, as _probably_ `Exception` is meant everywhere, but that's tedious.

Reply:
(Leaving this alone.)